testdata/parenthesis.go

```diff
@@ -16,3 +16,8 @@ const (
 	// Light best summon
 	Light Summon = "lucifer"
 )
+
+// Light2 best summon
```

Comment:
Why this?

Reply:
A test for already have the comment on the parens, then we do not need to add any comment